packages/core/expressions/cypress.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
```

Comment:
Is this file still needed?

Reply:
It looks like the file was removed already?

Reply:
Yes, removed